index.html

```diff
@@ -13,7 +13,7 @@
         <img src="icons/Icon - Menu.svg" alt="" />
       </nav>
       <section class="container-intro">
-        <h1>Hey there.<br />I’m Victor.<br /></h1>
+        <h1>Hey there.<br/>I’m Victor.<br/></h1>
         <p>
           I can help you build a product , feature or website Look through some
           of my work and experience! If you like what you see and have a project
```

Comment:
https://github.com/victormc13/Portfolio-project/blob/6fdaa8248a8121ad63c10cdbf3154bf7236a96d4/style.css#L88

- [ ] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` then assigning it wherever possible.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```